source/dotnet/Library/AdaptiveCards.Rendering.Html/AdaptiveCardRenderer.cs

```diff
@@ -446,17 +448,22 @@ protected static HtmlTag FactSetRender(AdaptiveFactSet factSet, AdaptiveRenderCo
                     Color = context.Config.FactSet.Value.Color,
                     Weight = context.Config.FactSet.Value.Weight,
                     IsSubtle = context.Config.FactSet.Value.IsSubtle,
+                    Wrap = context.Config.FactSet.Value.Wrap,
+                    // MaxWidth is not supported on the Value of FactSet. Do not set it.
                 };
                 var uiValue = context.Render(factValue)
                     .AddClass("ac-factvalue");
 
                 // create row in factset 
                 var uiRow = uiFactSet
                     .AddBodyRow();
+                uiRow.Style("height", "1px");
```

Comment:
What is this for?

Reply:
This has the facts be top aligned instead of centered aligned. 

---
In reply to: [185054118](https://github.com/Microsoft/AdaptiveCards/pull/1388#discussion_r185054118) [](ancestors = 185054118)